pkg/restic/commands.go

```diff
@@ -1,18 +1,22 @@
 package restic
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"time"
 
 	"github.com/appscode/go/log"
 	"github.com/appscode/stash/apis/stash/v1alpha1"
-	"github.com/pkg/errors"
 )
 
 const (
-	Exe = "/bin/restic_0.9.4"
+	ResticCMD = "/bin/restic"
```

Comment:
We must keep Stash backward compatible. `restic-0.8.3` is installed as `/bin/restic`. We can't change it. We must use different name for `restic-0.9.x` i.e. `/bin/restic_0.9.4`.